frontend/cypress/perf/common.ts

```diff
@@ -40,7 +39,15 @@ export function measureListsLoadTime(name: string, visits: number, listUrl: stri
     .then(() => {
       sum = sum / visitsArray.length;
 
-      const contents = `${name} list load time: ${(sum / 1000).toPrecision(5)} seconds\n`;
+      const contents = `${name} load time: ${(sum / 1000).toPrecision(5)} seconds\n`;
       cy.writeFile(reportFilePath, contents, { flag: 'a+' });
     });
 }
+
+export function measureListsLoadTime(name: string, visits: number, listUrl: string): void {
```

Comment:
```suggestion
export const measureListsLoadTime = (name: string, visits: number, listUrl: string): void => {
```
To be consistent with the rest of application code, prefer using arrow functions.